session-start-local.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -eu
+
+set -o allexport
+source .env
```

Comment:
Is there meant to be a `.env` file?

Reply:
you add the .env file in yourself locally but its never checked in

Reply:
Ah yep. I see that now